channeldb/graph.go

```diff
@@ -1905,13 +1969,57 @@ func delChannelEdge(edges, edgeIndex, chanIndex, zombieIndex,
 	)
 }
 
+// BatchedUpdateEdgePolicy updates the edge routing policy for a single directed
+// edge within the database for the referenced channel. The `flags` attribute
+// within the ChannelEdgePolicy determines which of the directed edges are being
+// updated. If the flag is 1, then the first node's information is being
+// updated, otherwise it's the second node's information. The node ordering is
+// determined by the lexicographical ordering of the identity public keys of the
+// nodes on either side of the channel.
+func (c *ChannelGraph) BatchedUpdateEdgePolicy(edge *ChannelEdgePolicy) error {
+	var (
+		isUpdate1    bool
+		edgeNotFound bool
+	)
+	return c.chanScheduler.Execute(&batch.Request{
+		Update: func(tx *bbolt.Tx) error {
```

Comment:
Is it an option to add two `error` return parameters, to prevent this pattern with the local variable?